docs/getting-started/installation/index.md

```diff
@@ -0,0 +1,85 @@
+# Installing JsTz
+
+Installation is still quite manual at this stage. Plus most of us started with all the right things installed to begin with. 
+If anything doesn't work we'd love to hear from you. (Actually we'll hate it. We're only human and we're like fixing people's builds as much as anybody else, but we grudgingly accept it will make the product better so we'll at least try to feign enthusiasm).
```

Comment:
while really funny,  I think such jokes could be easily misinterpreted by people who don't know the writer.